src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -224,9 +224,12 @@ def run_gridscan(
     yield from set_fast_grid_scan_params(fgs_motors, parameters.experiment_params)
     yield from wait_for_fgs_valid(fgs_motors)
 
+    # Start fake_stage here (async arming)
+    yield from bps.abs_set(fgs_composite.eiger, 1, group="arming")
```

Comment:
Must: This will call the `set`method n the `eiger` object, you want to call the `set` on the specific arming signal